app/controllers/mixins/checked_id_mixin.rb

```diff
@@ -104,4 +104,36 @@ def find_record_with_rbac_flash(model, id, resource_name = nil)
 
     checked_object
   end
+
```

Comment:
@martinpovolny Sorry, just noticed the methods are not inside the module block, can you please fix it?